interpreter/interpreter_test.go

```diff
@@ -155,15 +238,18 @@ func TestInterpreter_LogicalOrEquals(t *testing.T) {
 	provider := types.NewProvider(&exprpb.Expr{})
 	i := NewStandardInterpreter(packages.NewPackage("test"), provider)
 	interpretable := i.NewInterpretable(program)
-	result, _ := interpretable.Eval(
+	result, state := interpretable.Eval(
 		NewActivation(map[string]interface{}{
 			"a": true,
 			"b": "b",
 		}))
+	rhv, _ := state.Value(4)
```

Comment:
Please add a comment about where the numeric indices provided to `Value()` calls come from (here and above)

Reply:
done